### ISSUE TITLE
Controls don't respond in added games

### ISSUE THREAD

**Issue by DShiznit** [pathartl/BleemSync#383]
I was able to install BleemSync and add games to it just fine, no brownouts, using an ONN (dirt cheap Walmart store brand) 64gb USB 2.0 flash drive. Stock games work great, although the first time I run one after powering on takes upwards of 2 minutes to load. Games I've added however, while they do start, do not respond to the start button. They play opening cutscenes, initially very choppily I might add, then go to a "press start" screen at which point I can't do anything. It appears the D-Pad works, as I can switch menu items in Medal of Honor, but I can't select anything. I think the shoulder buttons also work, as I could skip the demo in Spyro by pressing all the shoulder buttons. It appears neither the start, nor the X button (and presumably other face buttons) are registering. 

I tried Spiderman, Spyro the Dragon, Army Men Sarge's Heroes, and Medal of Honor. 
The stock versions of Grand Theft Auto, Rayman, and Metal Gear Solid all worked. 

Attached are all my log files.
[bleemsync.log](https://github.com/pathartl/BleemSync/files/3178030/bleemsync.log)
[bleemsync_ui.log](https://github.com/pathartl/BleemSync/files/3178031/bleemsync_ui.log)
[bootloader.log](https://github.com/pathartl/BleemSync/files/3178032/bootloader.log)
[ui_menu.log](https://github.com/pathartl/BleemSync/files/3178033/ui_menu.log)

**Comment by DShiznit** [comment]
P.S. it also appears an empty game somehow was written to the stick by mistake. An empty game shows up in the menu, and there's an empty, undeletable folder on the flash drive (games/28) which is referenced in the log as an i/o error. I don't think it's related, but I thought I'd mention it to explain that error.

I'm running a disk check on the drive now as this leads me to suspect corruption (the unit did cycle on and off several times when I initially plugged it into my computer, until I adjusted the usb cable at which point it stabilized). I'll also try this again with a different flash drive and a smaller number of games, although I'd really like to be able to use this 64gb one.

**Comment by DShiznit** [comment]
Just finished starting from scratch (uninstalled with the uninstall image, reinstalled from a freshly formatted usb, added a handful of games via the web interface). The unit took it's sweet time to power down (I wound up pressing power 3 times over the course of a minute or 2) and when I ran chkdsk on the the usb afterward there was bitmap corruption, which is troubling, but there were no problems reported with the files themselves (unlike before). After starting it up everything seems to be working fine now. 

You guys can mark this resolved, but I'd recommend adding warnings to the installation and use instructions advising users to ALWAYS power down using the power button and WAIT, and NEVER disconnect power while the LED is green, even if the unit appears frozen. I did this several time before, not knowing any better, and I suspect this is what caused the drive to corrupt, and apparently my issues. I'd also recommend adding a quick chkdsk to the troubleshooting instructions, and I'd further recommend running chkdsk every time the unit is disconnected from power while the LED is green, and every time new games are added.
